lib/cli/cli.js

```diff
@@ -98,15 +98,17 @@ CLI.prototype.run = function(environment) {
 
     var PlatformChecker = require('../utilities/platform-checker');
     var platform = new PlatformChecker(process.version);
-    if (!platform.isValid && !this.testing) {
+    var recommendation = ' We recommend that you use the most-recent "Active LTS" version of Node.js.';
+
+    if (!this.testing) {
       if (platform.isDeprecated) {
         this.ui.writeDeprecateLine('Node ' + process.version +
-                                   ' is no longer supported by Ember CLI. Please update to a more recent version of Node');
+                                   ' is no longer supported by Ember CLI.' + recommendation);
       }
 
-      if (platform.isUntested) {
+      if (!platform.isTested) {
         this.ui.writeWarnLine('Node ' + process.version +
-                              ' has currently not been tested against Ember CLI and may result in unexpected behaviour.');
+                              ' is not tested against Ember CLI on your platform.' + recommendation);
```

Comment:
We've always had the distinction, but I've made it "useful" now.

Then:
- You receive the "untested" warning if your version is greater than the current hardcoded value.
- You receive the "deprecated" warning if your version was lower than the current hardcoded value.

Now:
- You receive the deprecated warning when your version is no longer supported per the `engines` key.
- You receive the untested warning when your version is not tested on your platform. This catches all unsupported Node versions in addition to the undeprecated but untested future releases.

If we keep both messages this is how we should do it. I like that it encourages people to stick to the happy path. It'll be noisy for a bit, and then it will calm down as people start to coalesce to a smaller set of versions.